### PR TITLE
Added social links and removed Head from designations

### PR DESCRIPTION
**Pull request by suneel87**
Signed-off-by: Suneel Nadipalli <nsuneel89@gmail.com>

#46 
-Added social media links
-Removed LinkedIn and replaced with YouTube
-Used new logos for social media links
-Removed "Head" from the end of designations in the Members page

### CODE REVIEW

**Comment by suneel87** [comment]
@aravindvnair99 I've made the changes only to members.ejs and footer.ejs. Hopefully, it should be a better PR this time :)

**Review by VinayakTekade — APPROVED** [review]
Made a few tweaks to the style. Great work.

File: functions/views/components/footer.ejs [hunk 1]
```diff
@@ -60,140 +60,153 @@
 						document.write(new Date().getFullYear());
 					</script>
 					<span>AYKYA</span>&nbsp;|&nbsp;All Rights
-					Reserved&nbsp;|&nbsp;Made with&nbsp;<i
-						class="fa fa-heart"
-						style="color: red"
-						aria-hidden="true"
+					Reserved
+					</br>
```
Comment (aravindvnair99) [inline functions/views/components/footer.ejs:64]: @VinayakTekade Void elements do not have end tags.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@suneel87 Don't use `alt` for `a`. Only these are allowed:

- [The `a` element | HTML Standard](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element)
- [Global Attributes | HTML Standard](https://html.spec.whatwg.org/multipage/dom.html#global-attributes)

Also, refrain from changing the formatting.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@suneel87 As you see, the size changed from XL to M. That's a big difference. Also, the padding of `70px` isn't required for laptops. Makes it look very unusual. And the members don't look aligned well on mobile devices. Some are vertically elongated rectangle and others square.

**Pull request merged**